### PR TITLE
Update dbcontext-creation.md (Issue 3064)

### LINKED ISSUE #3064: Machine translation should ignore examples of code
Page: https://docs.microsoft.com/en-us/ef/core/cli/dbcontext-creation?tabs=dotnet-core-cli
Line: Both IDesignTimeDbContextFactory.CreateDbContext and Program.CreateHostBuilder accept command line arguments.

IDesignTimeDbContextFactory.CreateDbContext is example of code.
But it is not marked as example of code, so the translator tries to translate it.

Can I submit a pull request that will fix it?

---
#### Сведения о документе

⚠ *Не вносите правки в этот раздел. Это необходимо для связывания страницы сайта docs.microsoft.com с вопросом на GitHub.*

* ID: 459d20e7-ba25-c162-5721-359ef3562999
* Version Independent ID: d988e806-cb86-19f5-7f76-6ce3bfcd94ed
* Content: [Создание DbContext во время разработки — EF Core](https://docs.microsoft.com/ru-ru/ef/core/cli/dbcontext-creation?tabs=dotnet-core-cli#feedback)
* Content Source: [entity-framework/core/cli/dbcontext-creation.md](https://github.com/dotnet/EntityFramework.Docs.ru-ru/blob/live/entity-framework/core/cli/dbcontext-creation.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @bricelam
* Microsoft Alias: **avickers**

### PR DESCRIPTION
**Pull request by AndrewKitu**
Fixes #3064

### CODE REVIEW

File: entity-framework/core/cli/dbcontext-creation.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ You can also tell the tools how to create your DbContext by implementing the `ID
 > This is fixed in EFCore 5.0 and any additional design-time arguments
 > are passed into the application through that parameter.
 
-A design-time factory can be especially useful if you need to configure the DbContext differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
+A design-time factory can be especially useful if you need to configure the `DbContext` differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
 
 ## Args
 
-Both IDesignTimeDbContextFactory.CreateDbContext and Program.CreateHostBuilder accept command line arguments.
+Both `IDesignTimeDbContextFactory.CreateDbContext` and `Program.CreateHostBuilder` accept command line arguments.
```
Comment (roji) [inline entity-framework/core/cli/dbcontext-creation.md:46]: As per https://github.com/dotnet/EntityFramework.Docs/issues/3064#issuecomment-771760832, these should use <xref> to reference the API docs.

Comment (AndrewKitu) [reply entity-framework/core/cli/dbcontext-creation.md]: fixed

File: entity-framework/core/cli/dbcontext-creation.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ You can also tell the tools how to create your DbContext by implementing the `ID
 > This is fixed in EFCore 5.0 and any additional design-time arguments
 > are passed into the application through that parameter.
 
-A design-time factory can be especially useful if you need to configure the DbContext differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
+A design-time factory can be especially useful if you need to configure the `DbContext` differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
 
 ## Args
 
-Both IDesignTimeDbContextFactory.CreateDbContext and Program.CreateHostBuilder accept command line arguments.
+Both <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory-1.CreateDbContext> and `Program.CreateHostBuilder` accept command line arguments.
```
Comment (roji) [inline entity-framework/core/cli/dbcontext-creation.md:46]:
```suggestion
Both <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory.CreateDbContext?displayProperty=nameWithType> and `Program.CreateHostBuilder` accept command line arguments.
```

File: entity-framework/core/cli/dbcontext-creation.md [hunk 3]
```diff
@@ -39,11 +39,11 @@ You can also tell the tools how to create your DbContext by implementing the `ID
 > This is fixed in EFCore 5.0 and any additional design-time arguments
 > are passed into the application through that parameter.
 
-A design-time factory can be especially useful if you need to configure the DbContext differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
+A design-time factory can be especially useful if you need to configure the `DbContext` differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
 
 ## Args
 
-Both IDesignTimeDbContextFactory.CreateDbContext and Program.CreateHostBuilder accept command line arguments.
+Both <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory.CreateDbContext?displayProperty=nameWithType> and `Program.CreateHostBuilder` accept command line arguments.
```
Comment (roji) [inline entity-framework/core/cli/dbcontext-creation.md:46]:
```suggestion
Both <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory.CreateDbContext%2A?displayProperty=nameWithType> and `Program.CreateHostBuilder` accept command line arguments.
```

File: entity-framework/core/cli/dbcontext-creation.md [hunk 4]
```diff
@@ -39,11 +39,11 @@ You can also tell the tools how to create your DbContext by implementing the `ID
 > This is fixed in EFCore 5.0 and any additional design-time arguments
 > are passed into the application through that parameter.
 
-A design-time factory can be especially useful if you need to configure the DbContext differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
+A design-time factory can be especially useful if you need to configure the `DbContext` differently for design time than at run time, if the `DbContext` constructor takes additional parameters are not registered in DI, if you are not using DI at all, or if for some reason you prefer not to have a `CreateHostBuilder` method in your ASP.NET Core application's `Main` class.
 
 ## Args
 
-Both IDesignTimeDbContextFactory.CreateDbContext and Program.CreateHostBuilder accept command line arguments.
+Both <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory.CreateDbContext%2A?displayProperty=nameWithType> and `Program.CreateHostBuilder` accept command line arguments.
```
Comment (roji) [inline entity-framework/core/cli/dbcontext-creation.md:46]:
```suggestion
Both <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory%601.CreateDbContext%2A?displayProperty=nameWithType> and `Program.CreateHostBuilder` accept command line arguments.
```

**Comment by roji** [comment]
Thanks @AndrewKitu!

**Pull request merged**